AutoGraphTests/WebSocketClientTests.swift

```diff
@@ -58,6 +61,83 @@ class WebSocketClientTests: XCTestCase {
         XCTAssertEqual(film?.remoteId, "ZmlsbXM6MQ==")
     }
     
+    func testUnsubscribeRemovesSubscriptions() {
+        let request = try! SubscriptionRequest(request: FilmSubscriptionRequest(), operationName: "film")
+        let subscriber = self.subject.subscribe(request: request, responseHandler: SubscriptionResponseHandler(completion: { _ in }))
+        
+        XCTAssertTrue(self.subject.subscriptions["film}"]!.contains(where: {$0.key == subscriber }))
+        
+        try! self.subject.unsubscribe(subscriber: subscriber)
+        
+        XCTAssertEqual(self.subject.subscriptions.count, 0)
```

Comment:
we should add an additional subscription with the exact same request and operation name so we know we're only clearing exactly the one with the matching uuid.

also we need to assert that the right request sends out to close it.